### PR TITLE
upgrade CC CoC to version 2.0

### LINKED ISSUE #427: update the Code of Conduct
Contributor Covenant’s current version is [2.0](https://www.contributor-covenant.org/version/2/0/code_of_conduct/).

### PR DESCRIPTION
**Pull request by retokromer**
## Checklist

* [x] I've referred to the [Guidelines for contributing](https://github.com/amiaopensource/ffmprovisr/blob/gh-pages/readme.md#guidelines-for-contributing)

Resolves https://github.com/amiaopensource/ffmprovisr/issues/427.

### CODE REVIEW

**Review by ablwr — APPROVED** [review]

**Comment by ablwr** [comment]
Seems good, and I'd like the other maintainers to weigh in. Since this is a change moving forward and not retroactive, it seems fine to add in (unlike, say, a license change).

File: code_of_conduct.md
Comment (privatezero) [inline code_of_conduct.md]: This looks generally good to me - I notice that the CoC violation link goes to the main AMIA reporting form. I think that as a resource that operates under the auspices of AMIA that this is good in many ways as it gives opportunity for enforcement at a higher level, however this does have the disadvantage of not alerting project maintainers who might be able to take more immediate action (at least within the smaller context of ffmprovisr and AMIA-OS). Thoughts?

Comment (ablwr) [reply code_of_conduct.md]: Oh that's a good point, I'm glad you caught that, Andrew. I don't have faith in AMIA providing an appropriate response here (as they have a history of claiming they receive no reports at all) and also believe it should fall to the project maintainers.

**Review by ablwr — CHANGES_REQUESTED** [review]
Agree with @privatezero that the method of reporting should not go to an AMIA form.

**Comment by kieranjol** [comment]
Aye great catch. Aside from Andrew's point, LGTM.

**Comment by kfrn** [comment]
Looks good. I agree about reports to come to us rather than be sent up the chain, so all G once that change is made.

Out of curiosity, how tied to the actual AMIA org is the AMIA opensource community? I know there the [Governance Interest Group](https://github.com/amiaopensource/governance), but otherwise, to me, it's always seemed pretty detached. Though that might be because I haven't been an AMIA member in a long time 😄

**Comment by retokromer** [comment]
This form is not my idea at all! I only followed @dericed’s [suggestion](https://github.com/amiaopensource/homebrew-amiaos/pull/178#issuecomment-471999292) for the `homebrew-amiaos` repository in this repository as well.

File: code_of_conduct.md
Comment (privatezero) [inline code_of_conduct.md:64]: This change is fine for me in terms of having this particular email address for myself  here - would want to confirm everyone else was ok with theirs being public before any merging though.

**Review by kieranjol — APPROVED** [review]

**Comment by ablwr** [comment]
It's okay for me. When all is done/approved, this should be squashed and merged as one commit.

**Review by ablwr — APPROVED** [review]

**Review by kfrn — APPROVED** [review]
All g with having my email in there

**Pull request merged**
